csrc/velox/CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ include_directories(velox)  # TODO: Do we need this? Or just ${VELOX_ROOT} is en
 # https://github.com/facebookexternal/presto_cpp/blob/f278b9b186d9753dc59f58f31bd09db1c89c3c2b/CMakeLists.txt#L104
 include_directories(${VELOX_ROOT})
 
+# Velox's thrid_party dependency
+include_directories(velox/third_party/googletest/googletest/include/)
```

Comment:
@ejguan : Is this the only way (have to add Velox's thrid_party dep include directory again... :( )